cmd/bank.go

```diff
@@ -51,16 +51,24 @@ func bankSendCmd() *cobra.Command {
 	return cmd
 }
 
-// ========== Query Functions ==========
+// ========== Querier Functions ==========
 
 func bankBalanceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "balance [key-or-address]",
+		Use:     "balances [key-or-address]",
 		Aliases: []string{"bal", "b"},
```

Comment:
I switched to `balances` to be more close with the Cosmos SDK binaries that use `balances`, also I think the idea of using it in the plural is because it returns all the tokens denoms, not only one (e.g. in Osmosis it would return the native token balances for  `uosmo` and `ion` and any other IBC token)